mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -139,15 +134,14 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
                                           startDate=startDateFirstYear,
                                           endDate=endDateFirstYear,
                                           calendar=calendar)
-        dsFirstYear = xr.open_mfdataset(
-            filesFirstYear,
-            preprocess=lambda x: preprocess_mpas(x,
-                                                 yearoffset=yearOffset,
-                                                 timestr='Time',
-                                                 onlyvars=variableList,
-                                                 varmap=variableMap))
-
-        dsFirstYear = remove_repeated_time_index(dsFirstYear)
+        dsFirstYear = open_multifile_dataset(file_names=filesFirstYear,
+                                             calendar=calendar,
+                                             time_variable_name='Time',
+                                             variable_list=variable_list,
+                                             variable_map=variableMap,
+                                             start_date=startDateFirstYear,
+                                             end_date=endDateFirstYear,
+                                             year_offset=yearOffset)
```

Comment:
This makes the usage much clearer, especially because additional selections via xarray aren't needed now.